### PR TITLE
Extract Fasterer::Config from Fasterer::FileTraverser

### PR DESCRIPTION
**Pull request by mmozuras**
Just a simple change that would make my work on [pronto-fasterer](https://github.com/mmozuras/pronto-fasterer/blob/master/lib/pronto/fasterer.rb) easier :smile:

### CODE REVIEW

File: lib/fasterer/config.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def file
+      @file ||= if File.exists?(FILE_NAME)
+        YAML.load_file(FILE_NAME)
+      else
+        nil_config_file
+      end
+    end
+
+    def nil_file
+      { SPEEDUPS_KEY => {}, EXCLUDE_PATHS_KEY => [] }
```
Comment (DamirSvrtan) [inline lib/fasterer/config.rb]: sorry for the EXTRA LATE response to this. However, i think this won't work - you're referencing a 'nil_config_file' and this methods name is 'nil_file'. I would love to see some test suite around this one.

**Comment by DamirSvrtan** [comment]
Thnx for this. I've also tried to make as little as possible changes to the outer interface so everything is backwards compatible. Tell me if there's any other modifications we could do to make pronto work easier.

**Comment by DamirSvrtan** [comment]
Released in 0.1.12.

**Comment by DamirSvrtan** [comment]
Note for clarity: I've added specs and fixed the issue with the code you've pushed.

**Comment by mmozuras** [comment]
@DamirSvrtan thanks! :smile:

**Pull request merged**